Hello Microverse Project Pull Request

YoutLone opened the pull request:
Hello Microverse Project : Module -1
In this project I :
  - Added html and css file on  htmlcss-branch

YoutLone committed:
Add html file

YoutLone committed:
Add css file

YoutLone committed:
fix stylelinc error

Shedrack-Sunday reviewed (changes requested):
Hello @YoutLone       👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 
![](https://user-images.githubusercontent.com/80612925/157397398-00904e26-60ec-46cd-9510-240661c250ef.gif)

## Highlights 
- Nice project setup✔️  
- Linters rightly setup ✔️
- Nice use of `.gitignore`✔️  
  
# Required Changes  🌵 

- [ ] I see you did great work, however, you're missing a very important part of the project which happens to be the [README FILE](https://github.com/microverseinc/readme-template). Kindly add it up with all sections included and customized.
- Only the `FAQ` part of the README file is optional 👍 

- [ ] Your pull request is not descriptive as it should be. But don't worry, you can follow [This link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) which has a very detailed way to make a professional pull request 👍 

## Section to edit PR:
![image](https://user-images.githubusercontent.com/50990925/234463302-1cf68f39-f1d8-4eac-b473-48e8789cb929.png)




# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).


https://github.com/microverseinc/readme-template

YoutLone commented:
Hello , After changing README.md only FAQ section is enough ? I want to
know in pull request need some questions or comment before submit request
review?

On Wed, 26 Apr 2023 at 10:04 AM Shedrack Sunday ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
>
> Hello @YoutLone <https://github.com/YoutLone> 👋
> CHANGES REQUIRED ♻️
>
> Great work on the project 👏
> You've done well implementing the requirements, However, there are a few
> you need to work on, to move to the next project. 💯
>
>
> <https://user-images.githubusercontent.com/80612925/157397398-00904e26-60ec-46cd-9510-240661c250ef.gif>
> Highlights
>
>    - Nice project setup✔️
>    - Linters rightly setup ✔️
>    - Nice use of .gitignore✔️
>
> Required Changes 🌵
>
>    -
>
>    I see you did great work, however, you're missing a very important
>    part of the project which happens to be the README FILE
>    <https://github.com/microverseinc/readme-template>. Kindly add it up
>    with all sections included and customized.
>    -
>
>    Only the FAQ part of the README file is optional 👍
>    -
>
>    Your pull request is not descriptive as it should be. But don't worry,
>    you can follow This link
>    <https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md>
>    which has a very detailed way to make a professional pull request 👍
>
> Section to edit PR:
>
> [image: image]
> <https://user-images.githubusercontent.com/50990925/234463302-1cf68f39-f1d8-4eac-b473-48e8789cb929.png>
> Optional suggestions
>
> Every comment with the [OPTIONAL] prefix won't stop the approval of this
> PR. However, I strongly recommend you take them into account as they can
> make your code better. Some of them were simply missed by the previous
> reviewer and addressing them will really improve your application.
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> Please, remember to tag me in your question so I can receive the
> notification.
>
> Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.
>
> As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have
> a limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.
>
> https://github.com/microverseinc/readme-template
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/YoutLone/Hello-Microverse-Project/pull/1#pullrequestreview-1401101545>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A4IWYLEMZQNJ36PHYXP7323XDCJUTANCNFSM6AAAAAAXLZGFHI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

Shedrack-Sunday commented:
Hello @YoutLone, kindly make changes by adding the README in this branch and push the changes then send the same PR for review

didierganthier reviewed (changes requested):
Hi @YoutLone ,

🎉 Great job on addressing the feedback from the previous review! You've made some solid progress in implementing the requested changes. However, there are still a few areas that need some attention before we can move on to the next project.

<img src="https://media.giphy.com/media/e56n4Z0mmkxZS/giphy.gif" />

## Highlights
- Added `.gitignore` ✔️ 
- Good linters setup ✔️ 
- Correct GitHub flow ✔️ 

## Required Changes 🛠
- [ ] Well done on improving the PR 💯 but it still needs some work. You already have a meaningful title for your pull request but make sure to add a short summary of changes introduced in the pull request 👇🏽 
  
![pull_request_desc](https://user-images.githubusercontent.com/45739963/234561540-eef10f6d-9c67-40d1-8cba-00568b453a2a.gif)
- [ ] Nice job on the README 📖 but for now it's still on the `main` branch and still contains all the boilerplate, make sure to copy the README you have on the `main` to the `htmlcss-branch` and then edit it to replace the info with relevant info to your project 📁 remember to keep all the sections as only the `FAQ` is optional, you may feel free to also remove sections such as `Live Version` as this project isn't meant to be deployed. 🚀 

- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- N/A

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

YoutLone commented:
Hello ,Do  I need to copy README.md file from main branch to htmlcss-branch
?

On Wed, 26 Apr 2023 at 6:04 PM Didier Peran Ganthier <
***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> Hi @YoutLone <https://github.com/YoutLone> ,
>
> 🎉 Great job on addressing the feedback from the previous review! You've
> made some solid progress in implementing the requested changes. However,
> there are still a few areas that need some attention before we can move on
> to the next project.
>
>
> <https://camo.githubusercontent.com/c21fff7d889bd167e51cd7aa24eaf53dd99cc831b1de1871e67d0ee4ae176e19/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6535366e345a306d6d6b785a532f67697068792e676966>
> Highlights
>
>    - Added .gitignore ✔️
>    - Good linters setup ✔️
>    - Correct GitHub flow ✔️
>
> Required Changes 🛠
>
>    - Well done on improving the PR 💯 but it still needs some work. You
>    already have a meaningful title for your pull request but make sure to add
>    a short summary of changes introduced in the pull request 👇🏽
>
> [image: pull_request_desc]
> <https://user-images.githubusercontent.com/45739963/234561540-eef10f6d-9c67-40d1-8cba-00568b453a2a.gif>
>
>    -
>
>    Nice job on the README 📖 but for now it's still on the main branch
>    and still contains all the boilerplate, make sure to copy the README you
>    have on the main to the htmlcss-branch and then edit it to replace the
>    info with relevant info to your project 📁 remember to keep all the
>    sections as only the FAQ is optional, you may feel free to also remove
>    sections such as Live Version as this project isn't meant to be
>    deployed. 🚀
>    -
>
>    Please review the comments under the review for specific changes that
>    still need to be made.
>
> Optional Suggestions 💡
>
>    - N/A
>
> 🚀 Keep pushing forward, you're almost there! If you have any questions
> or need clarification on anything, feel free to leave a comment in the pull
> request thread and don't forget to tag me so I'll receive the notification.
>
> *Please note*
>
>    - To avoid confusion and delays, please do not open a new pull request
>    for re-reviews. Use the same pull request submitted for the first review,
>    unless otherwise instructed.
>
> ------------------------------
>
> As per the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561>, you
> have a limited number of reviews per project. If you feel that the code
> review was unfair, you can request a second opinion using this form
> <https://airtable.com/shrQAqnBwek5a0O0s>.
>
> Cheers and happy coding! 👏👏👏
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/YoutLone/Hello-Microverse-Project/pull/1#pullrequestreview-1401786210>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A4IWYLEDB7RJYJGQER4J2TDXDEB4DANCNFSM6AAAAAAXLZGFHI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

didierganthier commented:
@YoutLone yes but once you've done that you also need to replace the hardcoded info on it to add info relative to your project. Sorry for the late reply, I was out.

YoutLone commented:
Hello Didier, what does this mean hardcoded info and I can't catch what you
mean.So can you tell me detail because of I worry this is my last chance
for code review.

On Wed, 26 Apr 2023 at 11:34 PM Didier Peran Ganthier <
***@***.***> wrote:

> @YoutLone <https://github.com/YoutLone> yes but once you've done that you
> also need to replace the hardcoded info on it to add info relative to your
> project. Sorry for the late reply, I was out.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/YoutLone/Hello-Microverse-Project/pull/1#issuecomment-1523768008>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A4IWYLHY4CV3RGXVI45IPOTXDFIQ3ANCNFSM6AAAAAAXLZGFHI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

didierganthier commented:
@YoutLone Take a look at [this one](https://github.com/mohamedgamaleldin999999/The-Jimmy-Project/blob/hello-microverse-setup/README.md)

YoutLone committed:
Copy README.md file from main branch to htmlcss-branch

abdulhamiid commented on README.md:
```diff
@@ -1 +1,154 @@
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
```
Kindly consider removing the `Server` and the `Database` section as they are out of the scope of this particular project. 🛠

We are meant to include only the technologies we used to build our project in this section. 

Let's also update the `Client` section by removing `React.js`. the project was built with `HTML` and `CSS` we can add those. 💪👍

Example:
```
<details>
  <summary>Client</summary>
  <ul>
    <li>HTML</li>
    <li>CSS</li>
  </ul>
</details>
```

abdulhamiid commented on README.md:
```diff
@@ -1 +1,154 @@
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
- [ ] The LICENSE link is broken. kindly consider adding a license to your repository.
![image](https://user-images.githubusercontent.com/78251927/234748305-f6ab2554-8767-43c5-8181-e2209707e30d.png)
- [ ]  Public repositories on GitHub are often used to share open-source software. For your repository to be open source, you'll need to license it so that others are free to use, change, and distribute the software.
 
- [ ] You can check here on [Adding a license to a repository](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) 🤗🛠

abdulhamiid commented on README.md:
```diff
@@ -1 +1,154 @@
+
+# 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/YoutLone)
+- Twitter: [@twitterhandle](https://twitter.com/@ThanMyoHtet64860)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/than-myo-htet-618b2524b)
```
![image](https://user-images.githubusercontent.com/78251927/234748934-bae3d278-f4bc-40d2-8a9e-cb96660baba9.png)

The link to your `Twitter` profile is broken, please kindly fix that 👍. Let's avoid broken links in our readme file for a professional repository. 🛠🤗

abdulhamiid commented on README.md:
```diff
@@ -1 +1,154 @@
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
```
As requested by the previous reviewer it is important that we customize every section of the [readme TEMPLATE](https://github.com/microverseinc/readme-template). to make our readme well-detailed and stand out from a bunch of others. 👌

Kindly add the `Future Features`, including 1-3 potential features to be added. 😊👍

abdulhamiid reviewed (changes requested):
# Status: Required Changes :recycle:
Hi @YoutLone 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, I really appreciate the effort you put in! but there are still some which haven't been addressed yet.

![almost-there](https://user-images.githubusercontent.com/57562869/143511358-9cde7daf-4ad0-4db2-8f15-18b54288ff91.gif)

## Highlights 🌟

- ✔️ Descriptions are descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

YoutLone committed:
Update README.md

YoutLone committed:
Update README.md

YoutLone committed:
Update README.md

YoutLone committed:
Update README.md

YoutLone committed:
Add License.md file to htmlcss-branch

YoutLone committed:
Merge branch 'htmlcss-branch' of github.com:YoutLone/Hello-Microverse-Project into htmlcss-branch

YoutLone commented:
So, I need to do some of requirement and I will make new pull request for
LICENSE.md to my htmlcss-branch.And then I can submit for code review .

On Thu, 27 Apr 2023 at 9:46 AM Adio Abdulhamid ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
> Status: Required Changes ♻️
>
> Hi @YoutLone <https://github.com/YoutLone> 👋,
>
> Great work on making the changes requested by a previous reviewer 👏
> You've done well implementing some of the requested changes, I really
> appreciate the effort you put in! but there are still some which haven't
> been addressed yet.
>
> [image: almost-there]
> <https://user-images.githubusercontent.com/57562869/143511358-9cde7daf-4ad0-4db2-8f15-18b54288ff91.gif>
> Highlights 🌟
>
>    - ✔️ Descriptions are descriptive
>    - ✔️ Linters checks are passing
>    - ✔️ Use GitHub Flow
>    - ✔️ Meaningful commit messages.
>
> Required Changes ♻️
>
> *Please kindly check the comments under the review.*
> Optional suggestions 🤷‍♂️
>
> *Every comment with the [OPTIONAL] prefix is not crucial enough to stop
> the approval of this PR. However, I strongly recommend you take them into
> account as they can make your code better.*
>
> Cheers and Happy coding!👏👏👏
> Important Notes 📝
>
>    - Feel free to leave any questions or comments in the PR thread if
>    something is not 100% clear.
>    - Please, remember to tag me in your question so I can receive the
>    notification.
>
> *Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.*
> ------------------------------
>
> *As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
> limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
> ------------------------------
>
> In README.md
> <https://github.com/YoutLone/Hello-Microverse-Project/pull/1#discussion_r1178573933>
> :
>
> > +### Tech Stack <a name="tech-stack"></a>
> +
> +* Version Control System
> +  -Git Hub
> +
> +<details>
> +  <summary>Client</summary>
> +  <ul>
> +    <li><a href="https://reactjs.org/">React.js</a></li>
> +  </ul>
> +</details>
> +
> +<details>
> +  <summary>Server</summary>
> +  <ul>
> +    <li><a href="https://expressjs.com/">Express.js</a></li>
> +  </ul>
> +</details>
> +
> +<details>
> +<summary>Database</summary>
> +  <ul>
> +    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
> +  </ul>
> +</details>
>
> Kindly consider removing the Server and the Database section as they are
> out of the scope of this particular project. 🛠
>
> We are meant to include only the technologies we used to build our project
> in this section.
>
> Let's also update the Client section by removing React.js. the project
> was built with HTML and CSS we can add those. 💪👍
>
> Example:
>
> <details>
>   <summary>Client</summary>
>   <ul>
>     <li>HTML</li>
>     <li>CSS</li>
>   </ul>
> </details>
>
> ------------------------------
>
> In README.md
> <https://github.com/YoutLone/Hello-Microverse-Project/pull/1#discussion_r1178575606>
> :
>
> > +## 📝 License <a name="license"></a>
> +
> +This project is [MIT](./LICENSE) licensed.
>
>
>    -
>
>    The LICENSE link is broken. kindly consider adding a license to your
>    repository.
>    [image: image]
>    <https://user-images.githubusercontent.com/78251927/234748305-f6ab2554-8767-43c5-8181-e2209707e30d.png>
>    -
>
>    Public repositories on GitHub are often used to share open-source
>    software. For your repository to be open source, you'll need to license it
>    so that others are free to use, change, and distribute the software.
>    -
>
>    You can check here on Adding a license to a repository
>    <https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository>
>    🤗🛠
>
> ------------------------------
>
> In README.md
> <https://github.com/YoutLone/Hello-Microverse-Project/pull/1#discussion_r1178577352>
> :
>
> > +👤 **Author1**
> +
> +- GitHub: ***@***.***(https://github.com/YoutLone)
> +- Twitter: ***@***.******@***.***)
> +- LinkedIn: [LinkedIn](https://linkedin.com/in/than-myo-htet-618b2524b)
>
> [image: image]
> <https://user-images.githubusercontent.com/78251927/234748934-bae3d278-f4bc-40d2-8a9e-cb96660baba9.png>
>
> The link to your Twitter profile is broken, please kindly fix that 👍.
> Let's avoid broken links in our readme file for a professional repository.
> 🛠🤗
> ------------------------------
>
> In README.md
> <https://github.com/YoutLone/Hello-Microverse-Project/pull/1#discussion_r1178582410>
> :
>
> > +- [👥 Authors](#authors)
> +- [🔭 Future Features](#future-features)
>
> As requested by the previous reviewer it is important that we customize
> every section of the readme TEMPLATE
> <https://github.com/microverseinc/readme-template>. to make our readme
> well-detailed and stand out from a bunch of others. 👌
>
> Kindly add the Future Features, including 1-3 potential features to be
> added. 😊👍
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/YoutLone/Hello-Microverse-Project/pull/1#pullrequestreview-1403095388>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A4IWYLDEGNRSTVFDP3SBY2TXDHQJLANCNFSM6AAAAAAXLZGFHI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

abdulhamiid commented on LICENSE.md:
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 YoutLone
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
```
- [ ] Great job adding a `LICENSE` to your project 💯 👍

abdulhamiid commented on README.md:
```diff
@@ -1 +1,136 @@
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
- [ ] The link to the License is broken. please let's fix that  🛠

- [ ] If you notice, the name of your license file is `LICENSE.md` and not `LICENSE`. 👀
![image](https://user-images.githubusercontent.com/78251927/234776379-954514fe-4da2-4609-afd6-323a798396b9.png)

- [ ] To fix this, let's update the `path` i.e from `[MIT](./LICENSE)` to `[MIT](./LICENSE.md)` so that when the link is clicked, it takes us to the LICENSE 👍🤗

abdulhamiid reviewed (changes requested):
# Status: Required Changes :recycle:
Hi @YoutLone 👋, 

Great work on making the changes requested by the previous reviewers 👏 
You've done well implementing some of the requested changes, I really appreciate the effort you put in! but there are still some which haven't been addressed yet.

![](https://media.giphy.com/media/UdaSUYClr92cjWpeVx/giphy.gif)

## Highlights 🌟

- ✔️ Descriptions are descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

YoutLone committed:
Update README.md

Olamarx reviewed (changes requested):
Hi @YoutLone, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
<img src="https://media3.giphy.com/media/3qBeElnF0WVsNuDuj4/giphy.gif"/>

 ## Highlights
- Readme file professional ✔️

 ## Required Changes ♻️
_Check the comments under the review._
- [ ] Your project is complete, you just need to resolve the conflict so that the reviewer can see the linter checks.
![image](https://user-images.githubusercontent.com/86200356/234793392-494ed2d1-9ecf-4495-8389-d2d74c8f88cd.png)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

TuyishimireEric reviewed (approved):
# Status: APPROVED 🟢  
### Hi  @YoutLone 👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job on implementing the project requirements! 🥇 
![done](https://media.giphy.com/media/l1uga8ObPBnIBoXUQ/giphy.gif)

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉

## Highlights
✔ Descriptive PR 👍 
✔ The README file is professional and has all the relevant headers 🎆
✔ Made suggested changes👍 

## Optional requirements

- [ ] **[OPTIONAL]** I have noticed that while making the suggested changes, the linter are not checking 😨. Kindly always remember, the Linter should be set correctly for every repository, you may follow [this tutorial ](https://github.com/microverseinc/linters-config)for guidance on setting the Linters in your Repository 😎😎
   - Also Linter result should be green. ```AKA: Linter should report NO errors.``` _Linters can help to improve the overall code quality of your project. Linters analyze your code and can identify potential issues or bugs before they become a problem._
   
   ![image](https://user-images.githubusercontent.com/102757126/234803121-087ae1a5-60a9-4166-8b41-5efc03808ce9.png)


### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/enus/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
